pipeline/clinvar/clinvar.py

```diff
@@ -174,7 +174,7 @@ def __init__(self, element, debug=False):
         self.dateLastUpdated = None
         cs = element.find("ClinicalSignificance")
         if cs != None:
-            self.dateLastUpdated = cs.get("DateLastEvaluated")
+            self.dateLastUpdated = cva.get("DateUpdated")
```

Comment:
we determined that the DateUpdated field is the correct field for determining updates to individual clinvar submissions